sdk/tests/router/config/router_config_test.py

```diff
@@ -18,3 +18,17 @@ def test_set_router_config_with_invalid_routes(actual, new_routes, expected, req
     actual.routes = new_routes
     with pytest.raises(expected):
         actual.to_open_api()
+
+@pytest.mark.parametrize(
+    "actual,default_route_id,expected", [
+        pytest.param(
+            "generic_router_config",
+            "test-route-not-exists",
+            InvalidRouteException
+        )
+    ])
+def test_set_router_config_with_invalid_default_route(actual, default_route_id, expected, request):
```

Comment:
Extreme nit: can we rename `default_route_id` to something along the lines of `invalid_route_id` or `not_the_default_route_id` to make it a little clearer that the route in that variable is actually invalid?